index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/452c25fd08.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/style.css" />
+  <title>Landing Page</title>
+</head>
+<body>
+<header id="header">
+	<img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" alt="hello world">
+	<nav id="nav-bar"> 
+		 <ul>
+        <li>
+          <a class = "nav-link" href = "#features">Features</a>
+        <li>
+          <a class = "nav-link" href = "#how-it-works">How it Works</a>
+        <li>
+          <a class = "nav-link" href = "#pricing">Pricing</a>
+      </ul>
+	</nav>
+</header>
+<div class="form">
+  
+<form action="https://www.freecodecamp.com/email-submit">
+  <h2>Handcrafted, home-made masterpieces Get intouch </h2>
+    <input id="email" type="email" placeholder="Enter Your Email required">
+    <input id="submit" type="submit" value="Get Started">
+  </form>
+	</div>
+  <div class="main">
+    <div class="featured-wrapper">
+      <div class="featured-icon">
+       <i class="fas fa-battery-full"></i>
+      </div>
+        <div class="featured-text">
+          <h3>Quality Assurance</h3>
+          <p>Our trombones use the shiniest brass which is sourced locally.This will increase the longevity of your purchase.</p>
+        </div>
+    </div>
+    <!-- end -->
+    <div class="featured-wrapper">
+      <div class="featured-icon">
+       <i class="fas fa-business-time"></i>
+      </div>
+        <div class="featured-text">
+          <h3>Fast Shipping</h3>
+          <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+        </div>
+    </div>
+      <!-- end of premium -->
+     
+      <div class="featured-wrapper">
+      <div class="featured-icon">
+       <i class="fas fa-battery-full"></i>
+      </div>
+        <div class="featured-text">
+          <h3>Quality Assurance</h3>
+          <p>Our trombones use the shiniest brass which is sourced locally.This will increase the longevity of your purchase.</p>
+        </div>
+    </div>
+      <!-- end of fast shipping -->
+      <div class="featured-wrapper">
+      <div class="featured-icon">
+       <i class="fas fa-battery-full"></i>
```

Comment:
Hi, @smunozmo your icons are repeating, you could use some different ones. 😎

Reply:
- kindly phrase the comments politely, as suggestions instead of orders. For example:
  * This is a polite suggestion: *"I think it would be better to use semantic tags"*
  * This reads as an order: *"Use semantic tags"*
- please include an explanation, a "why", for each requested change. For example:
  * This is a well explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
  * This request lacks a "why": *Why don't you use an article tag here instead of a div?*
- can you look for one more issue and add it using an inline comment? maybe that suggestion in the summary can be transferred to an inline comment 😄